src/gui/Src/Gui/CallStackView.cpp

```diff
@@ -280,17 +270,10 @@ void CallStackView::renameThreadSlot()
     DbgCmdExec(QString("setthreadname %1, \"%2\"").arg(ToHexString(threadId)).arg(DbgCmdEscape(threadName)));
 }
 
-void CallStackView::loadSymbolsSlot()
-{
-    char module[MAX_MODULE_SIZE] = "";
-    if(DbgGetModuleAt(getCellUserdata(getInitialSelection(), ColFrom), module))
-        DbgCmdExec(QString("symdownload \"%0\"").arg(module));
-}
-
-void CallStackView::loadSymbolsSingleThreadSlot()
+void CallStackView::loadSymbolsForThreadSlot()
 {
     char module[MAX_MODULE_SIZE] = "";
-    duint firstRowForThread = 0;
+    duint firstRowForThread = 1;

```

Comment:
I had wrongly assumed that the user data thread-id field was being set for the thread header row as well, which lead to a bug here. It should now work properly. I could've sworn I tested that earlier but I guess it slipped through.